frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -212,10 +212,10 @@ const HostSummary = ({
               {deviceUser ? "My device" : titleData.display_name || "---"}
             </h1>
             <p className="last-fetched">
-              {`Last fetched ${humanHostDetailUpdated(
-                titleData.detail_updated_at
-              )}`}
-              &nbsp;
+              {titleData.detail_updated_at
+                ? humanHostTimeDiffWithDateTip(titleData.detail_updated_at)
+                : // &nbsp;
```

Comment:
Is the `&nbsp;` commented out?

Reply:
Indeed it is, thank you for catching all of these